pkg/python-pynacl

```diff
@@ -0,0 +1,22 @@
+[mirrors]
+https://files.pythonhosted.org/packages/61/ab/2ac6dea8489fa713e2b4c6c5b549cc962dd4a842b5998d9e80cf8440b7cd/PyNaCl-1.3.0.tar.gz
+
+[vars]
+filesize=3351016
+sha512=1762d7d9add6ca59ce83e394071861f78f27d0f06ad1481255ebb66023186127e1a67b49a2c2f51bcbd97bdd151a3a2adea099c47c2a9f456dda5e3b1508af50
+pkgver=1
+uchkurl=https://pypi.org/project/PyNaCl/
+
+[deps]
+python
+libsodium
+
+[build]
+
+echo "#!/bin/true" > ldconfig && chmod +x ldconfig
+export PATH="${PWD}:${PATH}"
+export CFLAGS="$optcflags -fstack-protector"
```

Comment:
why does a crypto lib need -fstack-protector ? (just curious)

Reply:
Because it still fails to build with the error:
 undefined reference to `__stack_chk_fail_local'
even with the creation of libssp_nonshared.a for i?86 in the musl recipe; and adding -fstack-protector seems to fix the issue

Reply:
ah, that must be because you built libsodium with -fstack-protector